src/Writers/HTMLWriter.jl

```diff
@@ -1686,7 +1686,10 @@ function mdconvert(text::AbstractString, parent; kwargs...)
     return DOM.Node(text)
 end
 
-mdconvert(vec::Vector, parent; kwargs...) = [mdconvert(x, parent; kwargs...) for x in vec]
+function mdconvert(vec::Vector, parent; kwargs...)
+    mapping = get(kwargs, :mapping, nothing)
```

Comment:
```suggestion
function mdconvert(vec::Vector, parent; mapping = nothing, kwargs...)
```

More idiomatic pehaps?